### PR TITLE
remaining changelog entries for 2.068.0

### PR DESCRIPTION
**Pull request by MartinNowak**
requires https://github.com/D-Programming-Language/phobos/pull/3536

### CODE REVIEW

File: changelog.dd [hunk 1]
```diff
@@ -2,10 +2,11 @@ Ddoc
 
 $(D_S D Change Log,
 
-$(VERSION 068.0, XXX ??, 2015, =================================================,
+$(VERSION 068.0, Aug 09, 2015, =================================================,
 
 $(BUGSTITLE Compiler Changes,
 $(LI $(RELATIVE_LINK2 fix14828-warning, $(RED Warning:) Change in helper function generation might cause link error.))
+$(LI $(RELATIVE_LINK2 profile-gc, The new `-profile=gc` switch allows to profile memory allocation.))
```
Comment (WalterBright) [inline changelog.dd]: grammar: s/allows to profile/enables profiling/

File: changelog.dd [hunk 2]
```diff
@@ -63,6 +66,28 @@ $(LI $(LNAME2 fix14828-warning, $(RED Warning:) Change in helper function genera
     )
 )
 
+$(LI $(LNAME2 profile-gc, The new `-profile=gc` switch allows to profile memory allocation.)
```
Comment (WalterBright) [inline changelog.dd]: grammar: 'allows to'

File: changelog.dd [hunk 3]
```diff
@@ -63,6 +66,28 @@ $(LI $(LNAME2 fix14828-warning, $(RED Warning:) Change in helper function genera
...
+        Example:
+            $(PRE
+bytes allocated, type, function, file:line
+     171	immutable(char)[] example.concat example.d:4
+      24	closure D main example.d:18
+      16	example.Klass D main example.d:20
+            )
+    )
+
+    $(P $(RED Note:) Currently GC allocations using the core.memory interface are not profiled.
```
Comment (WalterBright) [inline changelog.dd]: remove `Currently`

File: changelog.dd [hunk 4]
```diff
@@ -84,6 +109,74 @@ $(LI $(LNAME2 lex-only-unittest, `unittest` blocks no longer parsed unless `-uni
 
 $(BUGSTITLE Language Changes,
 
+$(LI $(LNAME2 rangified-functions, Many phobos functions were rangified.)
+
+    $(P Many phobos functions that allocated GC memory were changed to support
+        ranges or were complemented by an alternative supporting ranges.
+        Using ranges often allows to eliminate GC allocations.
```
Comment (WalterBright) [inline changelog.dd]: Grammar: s/allows to eliminate/eliminates/

**Comment by MartinNowak** [comment]
Updated

**Pull request merged**